tests/App/config/routing.yaml

```diff
@@ -1,13 +1,13 @@
 hwi_oauth_redirect:
```

Comment:
Can you make "new commit" that will change the test configuration from yaml to php? It would give more light how many changes it's needed.

Reply:
I don't think, that switching tests/App configs to php would be a good idea. If you are using symfony/flex you receives config files also as yaml. Yaml is basically more readable than php arrays of arrays.
We also can not take advantage of `Symfony\Config\*Config`classes. They are built dynamically only by Symfony 5.3 and above and placed in var/cache on live site and are outside of the IDE-scope during testing.

Reply:
Hmmm yes you cannot depend on autogenerated classes, you would need to add `class_exists()` checks.

About readability of yaml... it's same as readable as plain php for me, but I can live with it.